### DISCUSSION TITLE
Images seem to be compressed

### DISCUSSION THREAD

**Discussion by rohansaw** [obss/sahi#992] [Q&A] [answered]
My original dataset is 24GB in size. However, after running `sahi coco slice` with --slice_size 512 and --overlap_ratio 0.25, the resulting directory is 8.7GB in size. Is this behavior intended from the slicing script?

**Comment by Raozey** [comment]
@rohansaw  you check if the file format is the same?

**Comment by fcakyon** [comment, marked as answer]
Hey @rohansaw, you can set the extension type as png for higher quality slice images https://github.com/obss/sahi/blob/aa69f4857f49f81ea69a1b9668663bde64c8d287/sahi/slicing.py#L424
